spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/controller/StreamDeploymentController.java

```diff
@@ -329,9 +315,8 @@ private boolean isPartitionedConsumer(ModuleDefinition module,
 			Map<String, String> moduleDeploymentProperties,
 			boolean upstreamModuleSupportsPartition) {
 		return upstreamModuleSupportsPartition ||
-				(module.getParameters().containsKey(BindingProperties.INPUT_BINDING_KEY) &&
```

Comment:
oops, I looked at the wrong method `upstreamModuleHasPartitionInfo`at my IDE . yeah, `consumer` in this case.